flask_resty/view.py

```diff
@@ -241,7 +241,7 @@ def parse_request_data(self):
         :return: The deserialized request data.
         """
         try:
-            data_raw = flask.request.get_json()["data"]
+            data_raw = flask.request.get_json(silent=True)["data"]
```

Comment:
This preserves the prior behavior of expecting None if invalid json was passed:
https://flask.palletsprojects.com/en/2.1.x/api/#flask.Request.get_json